### PR TITLE
Fix broken livenessprobe version table

### LINKED ISSUE #324: livenessprobe version info broken
The version table for https://kubernetes-csi.github.io/docs/livenessprobe.html currently looks like this:

Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | Min K8s Version | Max K8s Version | --|--|--|--|--|-- livenessprobe v2.0.0 | release-2.0 | v1.0.0 | quay.io/k8scsi/livenessprobe:v2.0.0 | v1.13 | - livenessprobe v1.1.0 | release-1.1 | v1.0.0 | quay.io/k8scsi/livenessprobe:v1.1.0 | v1.13 | - Unsupported. | No 0.x branch. | v0.3.0 | quay.io/k8scsi/livenessprobe:v0.4.1 | v1.10 | v1.16

Looks like it may have broken in https://github.com/kubernetes-csi/docs/pull/290

CC @msau42

### PR DESCRIPTION
**Pull request by Madhu-1**
The fixes broken livenessprobe version table

Fixes #324

### CODE REVIEW

File: book/src/livenessprobe.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 **Status:** GA/Stable
 
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | [Min K8s Version](kubernetes-compatibility.md#minimum-version) | [Max K8s Version](kubernetes-compatibility.md#maximum-version) |
---|--|--|--|--|--
+--|--|--|--|--|--|--
 [livenessprobe v2.0.0](https://github.com/kubernetes-csi/livenessprobe/releases/tag/v2.0.0) | [release-2.0](https://github.com/kubernetes-csi/livenessprobe/tree/release-2.0) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | quay.io/k8scsi/livenessprobe:v2.0.0 | v1.13 | -
```
Comment (msau42) [inline book/src/livenessprobe.md:11]: All of these rows are missing an entry for "Max CSI version"

Comment (Madhu-1) [reply book/src/livenessprobe.md]: @msau42 Thanks for the review Added Max CSI version

**Comment by msau42** [comment]
/lgtm
/approve
Thanks!

**Pull request merged**